package.json

```diff
@@ -45,6 +45,7 @@
     "grunt-tslint": "^3.1.0",
     "grunt-typings": ">=0.1.5",
     "intern": "^3.2.3",
+    "sinon": "1.14.1",
```

Comment:
word of caution, be careful, there be beasts in them sinon versions... if you find something that works, pin it...